cli/unit-openapi/src/models/cert_bundle_chain_cert_validity.rs

```diff
@@ -0,0 +1,29 @@
+/*
+ * NGINX Unit 1.31.1
+ *
+ * NGINX Unit is a lightweight and versatile application runtime that provides the essential components for your web application as a single open-source server: running application code, serving static assets, handling TLS and request routing.   **Important**: Ufnit's API is designed to expose any part of its configuration as an addressable endpoint.  Suppose a JSON object is stored at `/config/listeners/`:   ```json { \"*:8080\": { \"pass\": \"applications/wp_emea_dev\" } } ```  Here, `/config/listeners/_*:8080` and `/config/listeners/_*:8080/pass` are also endpoints.  Generally, object options are addressable by their names, array items—by their indexes (`/array/0/`).    **Note**: By default, Unit is configured through a UNIX domain socket. To use this specification with OpenAPI tools interactively, [start](https://unit.nginx.org/howto/source/#source-startup) Unit with a TCP port as the control socket.
```

Comment:
...

Reply:
This and all the junk below is SDK auto-generated stuff. It is nice for users to have a checked in copy of pre-generated SDK code in order to have a known good working version. However, as you can see it does grow a bit crufty overtime unless you have CI jobs set up to update it.